x86_64/src/devcons.rs

```diff
@@ -14,7 +15,7 @@ impl Uart for Uart16550 {
 
 pub fn init() {
     Console::new(|| {
-        static mut UART: Uart16550 = Uart16550 { port: 0x3f8 };
-        unsafe { &mut UART }
+        static UART: SyncUnsafeCell<Uart16550> = SyncUnsafeCell::new(Uart16550 { port: 0x3f8 });
```

Comment:
Should the same change be made to x86_64 devcons.rs?

Reply:
I don't think we need to.

Reply:
Oh wait, you mean aarch64? Yeah, probably.

Reply:
actually I meant x86-64 - I didn't realise this was already it 🤦